tools/test_configs/ESP8266Interface.json

```diff
@@ -26,7 +62,7 @@
             "target.network-default-interface-type": "WIFI",
             "nsapi.default-wifi-ssid": "\"WIFI_SSID\"",
             "nsapi.default-wifi-password": "\"WIFI_PASSWORD\"",
-            "nsapi.default-wifi-security": "WIFI_SECURITY",
+            "nsapi.default-wifi-security": "WPA2",
```

Comment:
To keep thos configs compileable out of the box I had to change this setting.
Otherwise we get:
```
[Error] NetworkInterfaceDefaults.cpp@86,0:  #20: identifier "NSAPI_SECURITY_WIFI_SECURITY" is undefined
```